content/blog/12-02-2021-test.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Catch 'Em Predator
+image: https://media.wired.com/photos/598e35994ab8482c0d6946e0/master/w_2240,c_limit/phonepicutres-TA.jpghttps://media.wired.com/photos/598e35994ab8482c0d6946e0/master/w_2240,c_limit/phonepicutres-TA.jpg
+author: Danielle Woo
+date: 11/02/2021
+content: >-
+        Catch Em’ Predator Workshop is a side event under the Lapor Predator initiative (link to LP page) to promote dialogue about online safety and better equip the participants in dealing with potential and existing threats. Aware of the need of integrating comprehensive and age appropriate sexual education that addresses cultural aspects of the digital world like consent, notions of privacy and sexting, MAU has developed specific content for each targeted age group. The workshop is free of charge as we aim to elevate as many beneficiaries as we can when it comes to online safety.
+
+        The Catch Em’ Predator Workshop is addressed to children from 9 to 13 years old, teenagers from 14 to 18 years old and young adults from 18 to 30 years old, including children and youth from the refugee and stateless groups and from the B40 community. It is conducted on a bi-weekly basis starting from June 2021 till December 2021.
+
+---
+
```

Comment:
Everything before `---` is the 'frontmatter' aka the metadata. So stuff like title,. image, date will be there.

The actual blog post needs to be after the `---` aka the actual content. See: https://content.nuxtjs.org/writing/